modules/media/locale/media.pot

```diff
@@ -21,3 +21,137 @@ msgstr ""
 msgid "Media"
 msgstr ""
 
+msgid "File Name"
+msgstr ""
+
+msgid "Instrument"
+msgstr ""
+
+msgid "Uploaded By"
+msgstr ""
+
+msgid "Date Taken"
+msgstr ""
+
+msgid "Comments"
+msgstr ""
+
+msgid "Last Modified"
+msgstr ""
+
+msgid "File Type"
+msgstr ""
+
+msgid "CandID"
+msgstr ""
```

Comment:
Per previous discussions this key should be now in loris namespace and should be `DCCID` in place of CandID